### PR TITLE
Ensure value of expressions is Python int

### LINKED ISSUE #484: int64 not json serializable
Some faciliyt location model, a CPMpy expression, when calling `exp.value()` on it, it returned an int64 (which was not serializable)

Why do we create non-standard types? Actually we don't, but numpy does somewhere, and we go along with it...

@kostis-init can provide the model

### PR DESCRIPTION
**Pull request by IgnaceBleukx**
Fixes #484

### CODE REVIEW

File: cpmpy/expressions/core.py [hunk 1]
```diff
@@ -329,10 +329,11 @@ def __rmod__(self, other):
 
     def __pow__(self, other, modulo=None):
         assert (modulo is None), "Power operator: modulo not supported"
-        if other == 0:
-            return 1
-        elif other == 1:
-            return self
+        if is_num(other):
```
Comment (Wout4) [inline cpmpy/expressions/core.py:332]: Why?

File: cpmpy/solvers/ortools.py [hunk 2]
```diff
@@ -217,7 +217,9 @@ def solve(self, time_limit=None, assumptions=None, solution_callback=None, **kwa
 
             # translate objective
             if self.has_objective():
-                self.objective_value_ = self.ort_solver.ObjectiveValue()
+                ort_obj_val = self.ort_solver.ObjectiveValue()
+                assert int(ort_obj_val) == ort_obj_val, "Objective value should be integer, please report on github"
```
Comment (Wout4) [inline cpmpy/solvers/ortools.py:221]: Does ortools return numpy ints? Do we need this for other solvers as well?

File: cpmpy/expressions/core.py (hunk 1 above)
Comment (IgnaceBleukx) [reply cpmpy/expressions/core.py:332]: The test `if other == 0` will always go through when `other` is an expression (will return a new expression). Alternatively we can do `if other == 0 is True` but I think this is more clear

File: cpmpy/solvers/ortools.py (hunk 2 above)
Comment (IgnaceBleukx) [reply cpmpy/solvers/ortools.py:221]: No, it returns floats (but always `.0` of course as everything is int)

Comment (IgnaceBleukx) [reply cpmpy/solvers/ortools.py:221]: I checked for the other solvers, they all return `int` except for Gurobi, which is also patched now

File: cpmpy/expressions/core.py (hunk 1 above)
Comment (Wout4) [reply cpmpy/expressions/core.py:332]: oh wauw that's a pretty serieus bug actually :o

**Pull request merged**